src/main/java/core/basesyntax/impl/ReturnHandlerImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.impl;
+
+import dao.FruitDao;
+import dao.FruitDaoImpl;
+import model.FruitTransaction;
+import strategy.OperationHandler;
+
+public class ReturnHandlerImpl implements OperationHandler {
+    private FruitDao fruitDao = new FruitDaoImpl();
```

Comment:
It's better to use dependency injection for `FruitDao` instead of creating a new instance inside the handler. This will make your code more testable and adhere to the SOLID principles.